src/client/datascience/dataScienceSurveyBanner.ts

```diff
@@ -127,16 +122,7 @@ export class DataScienceSurveyBanner implements IJupyterExtensionBanner, IExtens
     }
 
     public async activate() {
```

Comment:
This was hooked up to an invalid api (onKernelStateChange is not hooked up anymore). I'm going to look at the other usage of that API to see if I can remove it entirely.